<launch>
    <arg name="plot" default="false"/>
    <param name="mast_distance" value="2"/>
    <node pkg="autosail" name="true_wind_node" type="true_wind_node.py" respawn="true"/>
    <node pkg="autosail" name="gps_to_ned" type="gps_to_ned.py" respawn="true"/>
    <include file="$(find autosail)/launch/path_planner.launch"/>
    <include file="$(find autosail)/launch/ctrl.launch"/>
    <node pkg="autosail" name="network_node" type="network_node.py" respawn="true"/>
    <node pkg="web_video_server" name="web_video_server" type="web_video_server" respawn="true"/>
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
    <node pkg="autosail" name="route_planner" type="route_planner_service.py" respawn="true"/>
</launch>
